bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -25,9 +25,6 @@
   {% endif %}
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
Agree that updating to `page_css` is the way to go for this PR.

I opened [bug 1186605](https://bugzilla.mozilla.org/show_bug.cgi?id=1186605) a couple days ago to audit LESS & JS bundles. Please make a note of this instance in that bug so we don't forget. :smile: